myokit/formats/sbml/_importer.py

```diff
@@ -67,21 +66,28 @@ def _convert_unit(self, unit):
     def info(self):
         return info
 
-    def model(self, path):
-
+    def model(self, path, bind_time=True):
+        """
+        Returns myokit model specified by the SBML fil provided.
```

Comment:
Maybe some like "Returns a :class:`myokit.Model` based on the SBML file provided.

Also would need some explanation of what `bind_time` does

Reply:
Will do, once we decided what to do with the time binding, see comment above.

Reply:
Done.